src/task.c

```diff
@@ -144,8 +144,8 @@ static void NOINLINE save_stack(jl_tls_states_t *ptls, jl_task_t *t)
 {
     if (t->state == done_sym || t->state == failed_sym)
         return;
-    volatile char *_x;
-    size_t nb = (char*)ptls->stackbase - (char*)&_x;
+    char *frame_addr = (char*)jl_get_frame_addr();
+    size_t nb = (char*)ptls->stackbase - frame_addr;
```

Comment:
replacing `ptls->stackbase` with `jl_stackbase` is appropriate for release-0.4, right?